docs/structured-streaming-kafka-integration.md

```diff
@@ -187,50 +305,68 @@ The following options must be set for the Kafka source.
 The following configurations are optional:
 
 <table class="table">
-<tr><th>Option</th><th>value</th><th>default</th><th>meaning</th></tr>
+<tr><th>Option</th><th>value</th><th>default</th><th>mode</th><th>meaning</th></tr>
 <tr>
   <td>startingOffsets</td>
-  <td>earliest, latest, or json string
+  <td>earliest, latest (streaming only), or json string
   {"topicA":{"0":23,"1":-1},"topicB":{"0":-2}}
   </td>
-  <td>latest</td>
+  <td>streaming=latest, batch=earliest</td>
```

Comment:
same as above, put the value quotes. its clearer if 
"latest" for streaming, 
"earliest" for batch